beacon-chain/rpc/eth/validator/validator_test.go

```diff
@@ -268,21 +267,38 @@ func TestGetProposerDuties(t *testing.T) {
 		}
 		vid, _, has := vs.ProposerSlotIndexCache.GetProposerPayloadIDs(11, [32]byte{})
 		require.Equal(t, true, has)
-		require.Equal(t, types.ValidatorIndex(12289), vid)
+		require.Equal(t, types.ValidatorIndex(9982), vid)
```

Comment:
This changed because state's slot changed from `5` to `params.BeaconConfig().SlotsPerEpoch`. The implementation will always fetch a slot at an epoch boundary, which is why I modified the slot in the test.